src/NuGet.Core/NuGet.LibraryModel/LibraryDependency.cs

```diff
@@ -62,7 +76,9 @@ public class LibraryDependency : IEquatable<LibraryDependency>
             bool autoReferenced,
             bool generatePathProperty,
             bool versionCentrallyManaged,
-            LibraryDependencyReferenceType libraryDependencyReferenceType)
+            string aliases,
+            LibraryDependencyReferenceType libraryDependencyReferenceType
+            )
```

Comment:
breaking change to public API. Please create an overload

and move the `)` to the end of the previous line 😀

Reply:
Reverted tthis change and made the other constructor private. We don't really use the constructors in our code.

Reply:
reverted this and added a new private constructor. Obsoleted all the non default ones.  
We don't even use this in our code.